portfolio/src/main/webapp/script.js

```diff
@@ -12,70 +12,70 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Opens the selected content.
- */
 
+// Function that opens the selected content.
 function openContent(evt, page) {
-  let i, content, navitem;
+  let i;
 
-  content = document.getElementsByClassName("content");
+  // Set all the pages display to none
+  const content = document.getElementsByClassName("content");
   for (i = 0; i < content.length; i++) {
```

Comment:
Can this be done using forEach or map?

Reply:
Changed to using forEach on the HTMLCollection